docs/products/clickhouse/reference/s3-supported-file-formats.rst

```diff
@@ -0,0 +1,17 @@
+File formats for the S3 table function in Aiven for ClickHouse®
+===============================================================
+
+The `S3 table function <https://clickhouse.com/docs/en/sql-reference/table-functions/s3>`_ allows you to select and insert data in S3-compatible storages. The S3 table function in Aiven for ClickHouse® can be used with the following file formats:
+
+* ``Arrow``
+* ``CSV``
+* ``JSON``
+* ``TSV``
+* ``Parquet``
+* ``ORC``
+* ``Avro``
```

Comment:
These are just the file formats I tested, we support a lot more than just this but should be good for the moment